lib/cryptoexchange/exchanges/hitbtc/market.rb

```diff
@@ -0,0 +1,8 @@
+module Cryptoexchange::Exchanges
+  module Hitbtc
+    class Market
+      NAME = 'hitbtc'
+      API_URL = 'http://api.hitbtc.com/api/1/public'
```

Comment:
I think the url should be `http://api.hitbtc.com/api/1` instead of `http://api.hitbtc.com/api/1/public` ?